src/api/getLogs.ts

```diff
@@ -0,0 +1,18 @@
+import { axios } from 'boot/axios'
+import { url } from '../shared/variable.shared';
+import { useUserStore } from 'src/stores/user';
+const getLogs = async (id: number): Promise<string[] | null> => {
+    const userStore = useUserStore();
+    try{
+        const {data} = await axios.get(`${url}/compilations/${id}/logs`, { headers: {
+            'Authorization': `Bearer${userStore.user.jwt}`
```

Comment:
To działa? Bo nie pamiętam czy tam spacji się nie daje że "Bearer {token}"

Reply:
Działa